### PR TITLE
ECF-RP-370: Update app and db params for staging

### PR DESCRIPTION
**Pull request by ethanmills**
### Context
We want staging to be as similar to prod as possible. These are the values we will start with in prod. It is especially important to change the DB now, since it is hard to upgrade from an unencrypted db to an encrypted one

### CODE REVIEW

File: terraform/workspace-variables/staging.tfvars [hunk 1]
```diff
@@ -5,9 +5,9 @@ app_environment = "staging"
...
 paas_space_name = "early-careers-framework-staging"
-paas_postgres_service_plan = "tiny-unencrypted-11"
+paas_postgres_service_plan = "small-11"
 paas_app_start_timeout = "180"
 paas_app_stopped = false
 paas_web_app_deployment_strategy = "blue-green-v2"
 paas_web_app_instances = 1
-paas_web_app_memory = 512
\ No newline at end of file
+paas_web_app_memory = 8192
```
Comment (amazimbe) [inline terraform/workspace-variables/staging.tfvars:13]: What are these based on? Or did you just pick what you deemed suitable?

Comment (ethanmills) [reply terraform/workspace-variables/staging.tfvars:13]: Asked @cfq. I think JC picked a number months ago

**Review by kicferk1 — APPROVED** [review]
I guess this has been talked about and approved, right? As in, which sizes we are going for?

File: terraform/workspace-variables/staging.tfvars (hunk 1 above)
Comment (ethanmills) [reply terraform/workspace-variables/staging.tfvars:13]: Are you just curious, or do you disagree with the number?

**Comment by ethanmills** [comment]
Yep, Mehmet gave these as initial values, then we monitor and adjust

File: terraform/workspace-variables/staging.tfvars (hunk 1 above)
Comment (amazimbe) [reply terraform/workspace-variables/staging.tfvars:13]: Just curious

Comment (amazimbe) [reply terraform/workspace-variables/staging.tfvars:13]: Will we have just one web server then?

Comment (ethanmills) [reply terraform/workspace-variables/staging.tfvars:13]: I think at least initially, yes. We're only going to have 2 users for the next month... Hopefully ramp up will be quite gradual, so we can be reactive to load. It's something that's easy to change, so I think we're ok to start small

File: terraform/workspace-variables/staging.tfvars [hunk 2]
```diff
@@ -5,9 +5,9 @@ app_environment = "staging"
 # Gov.UK PaaS
 paas_api_url = "https://api.london.cloud.service.gov.uk"
 paas_space_name = "early-careers-framework-staging"
-paas_postgres_service_plan = "tiny-unencrypted-11"
+paas_postgres_service_plan = "small-11"
```
Comment (ethanmills) [inline terraform/workspace-variables/staging.tfvars:8]: For reference:

```
small-11                        100GB Storage, Dedicated Instance, Storage Encrypted, Max 200 Concurrent Connections. Postgres Version 11. DB Instance Class: db.t3.small.
```

**Review by amazimbe — APPROVED** [review]

**Pull request merged**